authors/apps/authentication/views.py

```diff
@@ -1,10 +1,20 @@
+import jwt
+import os
+
+from django.template.loader import render_to_string
+from jwt import ExpiredSignatureError
 from rest_framework import status
-from rest_framework.generics import CreateAPIView, RetrieveUpdateAPIView
+from rest_framework.generics import \
+    (RetrieveUpdateAPIView, CreateAPIView, UpdateAPIView)
```

Comment:
There's really no need for the backslash escape character. To add a line break in the import statement could be done as follows:
```python
from rest_framework.generics import (
    RetrieveUpdateAPIView, CreateAPIView, UpdateAPIView
)
```

Reply:
Thanks @indungu ..I did not know that. Making the changes in a minute